.github/actions/setup-maven-cache/action.yaml

```diff
@@ -0,0 +1,77 @@
+---
+name: Setup Maven Cache
+
+
+description: Configured GHA cache for Maven global cache dir (no save on PRs), see https://github.com/camunda/camunda/wiki/CI-&-Automation#caching-strategy
+
+inputs:
+  java:
+    description: If true, will set up Java; defaults to true
+    required: false
+    default: "true"
```

Comment:
This input is unneeded and should be removed. All steps in this action are conditional on this input so it is easier to just not call the action if the caller identifies that no Java setup is desired.